src/components/cc-tcp-redirection/cc-tcp-redirection.types.d.ts

```diff
@@ -1,21 +1,21 @@
 interface TcpRedirection {
   namespace: string;
   isPrivate: boolean;
```

Comment:
question: I don't when this property is filled. Is it really needed?

Reply:
An issue has been created on the API side.

We can confirm that right now, this is not used at all within the console because the API does not provide this info in any way.

Reply:
(discussed on Slack) 
In the meantime we have decided to handle it within the smart so I'll take care of it